pkg/cmd/root/help.go

```diff
@@ -302,3 +302,28 @@ func dedent(s string) string {
 	}
 	return strings.TrimSuffix(buf.String(), "\n")
 }
+
+func buildAliases(cmd *cobra.Command, aliasList []string) string {
```

Comment:
I think we'll make our lives easier if we remove the string formatting from here and only build up the list of aliases, then format it with `strings.Join` like so:

```go
func buildAliasList(cmd *cobra.Command, aliases []string) []string {
	if !cmd.HasParent() {
		return aliases
	}

	parentAliases := append(cmd.Parent().Aliases, cmd.Parent().Name())
	sort.Strings(parentAliases)

	var aliasesWithParentAliases []string
	// e.g aliases = [ls]
	for _, alias := range aliases {
		// e.g parentAliases = [codespaces, cs]
		for _, parentAlias := range parentAliases {
			// e.g. aliasesWithParentAliases = [codespaces list, codespaces ls, cs list, cs ls]
			aliasesWithParentAliases = append(aliasesWithParentAliases, fmt.Sprintf("%s %s", parentAlias, alias))
		}
	}

	return buildAliasList(cmd.Parent(), aliasesWithParentAliases)
}
```

And at the call site:

```go
strings.Join(buildAliasList(cmd, cmd.Aliases), ", "))
```